dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -615,7 +612,7 @@ def test_frame_series_arithmetic_methods():
 
             assert_eq(l.rdiv(r, fill_value=0), el.rdiv(er, fill_value=0))
             assert_eq(l.rtruediv(r, fill_value=0), el.rtruediv(er, fill_value=0))
-            if not PANDAS_GT_0250:
+            if PANDAS_VERSION <= "0.25.0":
```

Comment:
This is only true when pandas is exactly equal to 0.25.0, or is it 0.25.x?

In general, I prefer the pattern

```python
if CONSTANT:
    ...
```

since it makes things easier to clean up later.

Reply:
Definitely. I was getting a little loosey-goosey with my replacements.